### DISCUSSION TITLE
Ensure inscriptions aren't accidentally sent

### DISCUSSION THREAD

**Discussion by skyrover7** [ordinals/ord#1552] [Q&A] [answered]
I see reference to this question in #1137 but just wanted some clarity on the steps required in order to ensure that a wallet holding inscriptions can safely send BTC without sending any existing inscriptions?

Thank you!

**Comment by raphjaph** [comment, marked as answer]
If you use `ord wallet` it will ensure that none of your inscriptions are sent unless explicitly specified. If you drop down into `bitcoin-cli` to send then you will most likely accidentally send ordinals because `bitcoin-cli` is not ordinal aware. There are some open issues to address this though: #1269 and #1147

**Reply by skyrover7** [reply]
thanks @raphjaph for the quick response!

I'm only using/referring to ord wallet. 

so if i sent 10000 sats to an address and inscribe a file. let's say it only uses 1000 sats. is there a way to move the balance sats in that address out of there?

**Reply by raphjaph** [reply]
inscribing attaches a default of 10_000 sats as postage to the inscribed sat. If you have 20_000 sats in your wallet to begin with you will be able to send the left over sats (non-postage aka cardinal). `ord wallet balance` shows you how many sats you have for that purpose.
